Makefile

```diff
@@ -43,6 +43,8 @@ SECRET_ACCESS_KEY_FILE := .kube-secrets/aws/secret_access_key.secret
 #########################################
 
 TOOLS_DIR := hack/tools
+# temporarily until Gardener version has been revendored >= v1.54
```

Comment:
This is not required now, the latest vendored gardener version is 1.56

Reply:
Still it doesn't harm to use a later golangci-lint version. We can remove it later.